openedx/stanford/djangoapps/auth_lagunita/models.py

```diff
@@ -48,3 +49,24 @@ def __unicode__(self):
             submitted_marketing_optin=self.submitted_marketing_optin,
         )
         return message
+
+
+def _need_subscribed():
+    """
+    Return a filtered list of users pending subscription to marketing list
+    """
+    infos = Info.objects.filter(
+        requested_marketing_optin=True,
+        submitted_marketing_optin=False,
+        user__is_active=True,
+    )
+    infos = infos.exclude(
+        Q(user__email='') |
+        Q(user__email=None) |
+        Q(user__email__endswith='@example.com') |
+        Q(user__email__endswith='.example.com')
+    )
+    return infos
+
+
+Info.objects.need_subscribed = _need_subscribed
```

Comment:
Django docs only mention creating an entire [separate custom manager](https://docs.djangoproject.com/en/2.2/topics/db/managers/#adding-extra-manager-methods) to handle custom queries, rather than directly modifying the default objects manager. Just curious the reason for the latter approach, is it just a one-off thing?